tracex_project/tracex/logic/utils.py

```diff
@@ -221,31 +219,25 @@ def get_events_df(query: Q = None):
                     "time:end_timestamp": event.end,
                     "time:duration": event.duration,
                     "attribute_location": event.location,
+                    "activity_relevance": getattr(
```

Comment:
I think this is unnecessary since you added metrics to the journey_comparisons and we setting default values as None for new extracted traces is a the metrics analyzer was not selected in the conifg
![image](https://github.com/bptlab/TracEX/assets/73845255/7312e3c9-f611-4aac-b771-e7b6e16a7d39)